src/auxsolvers/helmholtz_projector.cpp

```diff
@@ -52,9 +52,9 @@ HelmholtzProjector::Project(hephaestus::GridFunctions & gridfunctions,
 
   if (!fespaces.Has(_h1_fespace_name))
   {
-    std::cout << _h1_fespace_name + " not found in fespaces when "
-                                    "creating HelmholtzProjector. "
-                                    " Extracting from GridFunction\n";
+    logger.info(
+        "{} not found in fespaces when creating HelmholtzProjector. Extracting from GridFunction",
```

Comment:
Worth getting the class name (HelmholtzProjector) automatically rather than including it explicitly - easier to reuse this info block in other classes then, and more robust against name changes

Reply:
Done!